### DISCUSSION TITLE
Je ne parviens pas acceder au service de copilot avec mon compte Github

### DISCUSSION THREAD

**Discussion by Franckkoukougnon** [community/community#114580] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Je tente de m'abonner a copilot avec mon compte github , mais c'est impossible

**Comment by Akash1134** [comment]
Hi @Franckkoukougnon , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
